src/utils.ts

```diff
@@ -1,5 +1,6 @@
 const DEBUG_ENFLAKE = typeof process !== 'undefined' ? process.env?.DEBUG_ENFLAKE || false : false;
 export const PERCENT_CHANCE_OF_SUCCESS = typeof process !== 'undefined' ? +(process.env?.PERCENT_CHANCE_OF_SUCCESS || 98) : 98;
+export const FLAKE_CHANCE = 1 - PERCENT_CHANCE_OF_SUCCESS;
```

Comment:
Where are we using FLAKE_CHANCE?

Reply:
I didn't actually use it beyond the documentation, just seemed useful to have.

Reply:
oooo now that you mentioned "documentation"- don't forget to also modify the README to add your new change to the `list of havoc to be caused`